dockerhub-prune-tags.py

```diff
@@ -13,7 +13,7 @@
 
 def usage():
     script = os.path.basename(__file__)
-    print("Usage: {script} [opensciencegrid/]<REPO>".format(**locals()))
+    print(f"Usage: {script} [opensciencegrid/]<REPO>")
```

Comment:
Hey now, `.format(**locals())` is legit python3!  :)

Reply:
My apologies :)